examples/meta/src/kernel/const.sg.in

```diff
@@ -0,0 +1,18 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File testdat = csv_file("@SHOGUN_DATA@/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel const_kernel = kernel("ConstKernel", const_value=23.00)
```

Comment:
same here, could this not just be `23.0`?

Reply:
Done